query-engine/metrics/Cargo.toml

```diff
@@ -4,6 +4,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+futures = "0.3"
```

Comment:
Could `futures` become a `workspace` dependency? We use it in several crates already.